hyperswitch/PMMViewController.swift

```diff
@@ -0,0 +1,115 @@
+import Foundation
+import WebKit
+import Combine
+import SwiftUI
+
+class PaymentMethodManagementViewController: UIViewController {
+    @ObservedObject var hyperViewModel = HyperViewModel()
+    private var paymentSession: PaymentSession?
+    private var cancellables = Set<AnyCancellable>()
+    
+    private func setupPaymentWidget(topElement: UIView, onAddPaymentMethod: @escaping () -> Void) {
+        guard hyperViewModel.paymentSession != nil else { return }
+        
+        let paymentWidget = PaymentMethodManagementWidget(
+            ephemeralKey: PaymentSession.ephemeralKey ?? "",
+            frame: CGRect(x: 0, y: 0, width: view.bounds.width, height: view.bounds.height-160),
```

Comment:
is this needed, does it actually change the size?

Reply:
it does, it's required to define the view's bounds

Reply:
what happens if you give 1/2 height of VC, can you add a screenshot

Reply:
<img width="345" alt="image" src="https://github.com/user-attachments/assets/50287259-6cfb-4b88-ba90-a7974468a769">

_only white part of the screen is widget, while the blue part is merchants' view._

PMM widget's frame defined as:
```
let paymentWidget = PaymentMethodManagementWidget(
            frame: CGRect(x: 0, y: 0, width: view.bounds.width, **height: view.bounds.height/2**),
            ...
            )
```